Android/app/src/main/java/app/intra/DnsVpnService.java

```diff
@@ -53,13 +53,35 @@
   private static final String LOG_TAG = "DnsVpnService";
   private static final int SERVICE_ID = 1; // Only has to be unique within this app.
   private static final String CHANNEL_ID = "vpn";
+  private static final String NO_PENDING_CONNECTION = "This value is not a possible URL.";
```

Comment:
nit: maybe use a more standard sentinel value like `""` or `null`?

Reply:
Unfortunately that doesn't work here because `null` and `""` are both treated as meaning "default server".